public/data/rufilboy.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "Ilyas Rufai",
+  "type": "personal",
+  "bio": "DevOps Engineer & Open Source Enthusiast!",
+  "avatar": "https://avatars.githubusercontent.com/u/69714197?v=4",
```

It's best to use it this way so that the image updates automatically; plus, the link is shorter.
```suggestion
  "avatar": "https://github.com/rufilboy.png",
```